File: 05 - Matemática/1240 - Encaixa ou Não I/problem1240.py
	"""   
		URI Online Judge 
		Problema 1240 - Encaixa ou Não I
		https://www.urionlinejudge.com.br/repository/UOJ_1240.html
	"""
	casos = int(input())

	for i in range(casos):
		pot = 1
		"""
		Python 2.7 
		a, b = map(int, raw_input().split()) 
		"""

		a, b = input().split()	
		n = int(b)
		
		while (int(n)!=int(0)):
			n /= 10
			pot *= 10
		
		if int(a)%pot == int(b):
			print("encaixa")
		else:
			print("nao encaixa")

File: 05 - Matemática/1240 - Encaixa ou Não I/problem1240.cpp
/*  
	URI Online Judge 
	Problema 1240 - Encaixa ou Não I
    https://www.urionlinejudge.com.br/repository/UOJ_1240.html
*/

#include <iostream>
#include <cmath>
using namespace std;


int main() {
	
	int a, b, tam, n, casos, pot;
	scanf("%d", &casos);
	
	for (int i=0; i<casos; i++) {
		scanf("%d %d", &a, &b);
		tam = 0;
		n = b;
		while (n!=0){
			n /= 10;
			tam++;
		}
		pot = pow(10,tam);
		
		if (a%pot == b) {
			printf("encaixa\n");
		} else {
			printf("nao encaixa\n");
		}
	}
    return 0;
}

